pkgs/by-name/go/goshs/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+buildGoModule rec {
+  pname = "goshs";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "patrickhener";
+    repo = "goshs";
+    tag = "v${version}";
+    hash = "sha256-xq9BqWhUZfk4p5C6d5Eqh98bs0ZDTjpy5KnvhV/9Jew=";
+  };
+
+  vendorHash = "sha256-ECh0K3G6VAAJihqzzlWaEQclfXa0Wp/eFL16ABa7r+0=";
+
+  nativeInstallCheckInputs = [ versionCheckHook ];
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  doInstallCheck = true;
+
+  meta = {
+    description = "HTTP Server";
```

```suggestion
    description = "Simple yet feature-rich web server";
```
I feel like the description should be something more than just "HTTP Server"? I grabbed this line from their website